src/coap_mbedtls.c

```diff
@@ -32,6 +32,7 @@
  * MBEDTLS_SSL_CLI_C - defined for client side functionality
  * MBEDTLS_SSL_PROTO_DTLS - defined for DTLS support
  * MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED - defined if PSK is to be supported
+ * or MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED - defined if PSK is to be supported
```

Comment:
Here, the new line looks like it should have replaced the previous line.

Reply:
2.16.5 is built with MBEDTLS_KEY_EXCHANGE__SOME__PSK_ENABLED (double _)
2.25.0 is built with MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED (single _)

at some point things were changed so we need to recognize how things were built and whether the library was built for PSK or not.  

So, the description unfortunately is correct.

Reply:
Okay, thanks.